DJHelper/DJHelper/Model/Event Model/Event Convienience/Event+Convenience.swift

```diff
@@ -17,9 +17,6 @@ extension Event {
                      eventDescription: String,
                      eventDate: Date,
                      hostID: Int32,
-                     locationID: Int32,
```

Comment:
Good work getting your hands dirty and cleaning up all of the deleted properties. Must have been messy.